### PR TITLE
PR Review Fixes

### PR DESCRIPTION
**Pull request by pinkyrout**
Related to PR#2
Summary -
1. Used tailwind classes instead of custom css class.
2. Used nui table hover class and removed custom css file.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 1]
```diff
@@ -26,7 +25,7 @@ export default function NoteTable({
 
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx]: Please add `nui-table--hover` class too. 
It adds the hover effects for the table rows.

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@pinkyrout 
Please check the suggestion.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx (hunk 1 above)
Comment (pinkyrout) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx]: I have added the nui table hover class.

**Comment by karthiknmenon** [comment]
LGTM.

**Review by karthiknmenon — APPROVED** [review]

**Comment by karthiknmenon** [comment]
@pinkyrout As mentioned earlier, please add these to PR descriptions - 
1. If it's based on an issue then, `Fixes #issueNumber`
2. If it's based on a PR Review, then, `Related to PR#PRNumber`

**Pull request merged**